csfieldguide/templates/interactives/training-ground.html

```diff
@@ -0,0 +1,80 @@
+          <div id="game-area" class="col-7">
+            <div class="row">
+              <div class="col-4">{% trans 'Sticks remaining:' %} <span id="remaining-sticks"></span></div>
+              <div class="col-4">{% trans 'AI wins:' %} <span id="ai-wins"></span></div>
+              <div class="col-4">{% trans 'Games played:' %} <span id="games-played"></span></div>
+            </div>
+            <div id="sticks-area" class="row text-left"></div>
+            <div id="status-text" class="col-12 text-center"></div>
+            <button type="button" class="btn btn-success m-2" id="button_start">{% trans 'START' %}</button>
+            <div id="game-buttons" class="col-12 text-center d-none">
+              <button id="button_1" type="button" class="btn btn-success m-2">&nbsp;&nbsp;&nbsp;I&nbsp;&nbsp;&nbsp;</button>
+              <button id="button_2" type="button" class="btn btn-success m-2">&nbsp;&nbsp;I I&nbsp;&nbsp;</button>
+              <button id="button_3" type="button" class="btn btn-success m-2">&nbsp;I I I&nbsp;</button>
+            </div>
+            <div id="end-buttons" class="col-12 text-center d-none">
+              <button id="button_simulate" type="button" class="btn btn-primary m-2">{% trans 'Simulate' %}</button>
+              <button id="button_rematch" type="button" class="btn btn-primary m-2">{% trans 'Rematch' %}</button>
+            </div>
+            <div id="quit-buttons" class="col-12 text-right d-none">
+              <button id="button_quit" type="button" class="btn btn-danger m-2">{% trans 'Quit' %}</button>
+            </div>
+            <div id="splash-text" class="d-none"></div>
+            <button id="button_cancel" type="button" class="btn btn-danger btn-lg d-none">{% trans 'STOP' %}</button>
+          </div>
+          <table id="data-table" class="col-5"></table>
+        </div>
+      </div>
+      <div id="game-parameters">
+        <h3>{% trans 'Game parameters' %}</h3>
+        <div class="row">
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="num-simulations-select">{% trans 'Number of simulations:' %}</label>
+            <small id="num-sims-help" class="form-text text-muted">{% trans 'Nathaniel can practice before we start.' %}</small>
+            <div id="num-simulations-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="num-sticks-select">{% trans 'Number of sticks:' %}</label>
+            <small id="num-sticks-help" class="form-text text-muted">{% trans 'How many sticks in the pile?' %}</small>
+            <div id="num-sticks-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="sensitivity-select">{% trans 'AI sensitivity (%):' %}</label>
+            <small id="sensitivity-help" class="form-text text-muted">{% trans 'Slight adjustments or sweeping changes?' %}</small>
```

Comment:
Problem is it's not how quickly the AI learns, if the sensitivity is too high the AI will actually learn slower than somewhere in the middle of the range

Reply:
True. Perhaps if we decide to add some instructions/description of how the interactive works we could explain how the sensitivity works there? Then just have 'Slight changes or big changes' instead of sweeping? Thoughts?

Reply:
Going with `How much to change after each game?` for now